recipes/cortecs/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "cortecs" %}
+{% set version = "0.2.0b0" %}
```

Comment:
This one falls under https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Reply:
Thanks for letting me know! As the simplest solution, could I just remove the beta tag for this first conda release? I.e., just change line 2 to {% set version = "0.2.0" %} ?

Reply:
Sure. If you can make it a stable release, even better.

Reply:
Great. I'll keep it as 0.2.0 for now if that's alright — I'm making some changes soon that will bring to 1.0.0 (currently undergoing some peer review)